### ISSUE TITLE
Unable to find user photon

### ISSUE THREAD

**Issue by OhaDerErste** [Xyphyn/photon#115]
**Describe the bug**
Tried to run the container with the docker command from the readme( `docker run -p 8080:3001 -e PUBLIC_INSTANCE_URL=lemmy.ohaa.xyz ghcr.io/xyphyn/photon:latest`). This is the error I got:
`docker: Error response from daemon: unable to find user photon: no matching entries in passwd file.`
**To Reproduce** (if applicable)
1. Run the provided docker command
2. errors out  
  
**Expected behavior**

it shouldnt produce an error

**Comment by Xyphyn** [comment]
Fixed in v0.9.18

**Comment by OhaDerErste** [comment]
How do I get this version? Should I build it myself?

**Comment by OhaDerErste** [comment]
Tried building from git. Still the same error.

**Comment by Xyphyn** [comment]
Pull from the docker registry. The image was pushed a minute ago.

**Comment by OhaDerErste** [comment]
How would I do this exactly? Dont have much experience with docker

**Comment by Xyphyn** [comment]
From the README:

```
# (Optional) Use the env var PUBLIC_INSTANCE_URL to set the default instance URL.
docker run -p 8080:3000 -e PUBLIC_INSTANCE_URL=example.com ghcr.io/xyphyn/photon:latest
```

**Comment by OhaDerErste** [comment]
thats exactly the command I ran

**Comment by Xyphyn** [comment]
Try it again.

**Comment by OhaDerErste** [comment]
same error

**Comment by Xyphyn** [comment]
Strange. It works on my server and locally.

**Comment by OhaDerErste** [comment]
Got It to start with manually pulling the image again. Cant acess the website now

**Comment by Xyphyn** [comment]
Did you bind to the right port?

**Comment by OhaDerErste** [comment]
8080:3002. 3000 and 3001 are already used on my system

**Comment by Xyphyn** [comment]
You binded to port `3001` it looks like. You must bind to port `3000`.

the port to the left of the `:` is what the port will actually be on your machine.

`3002:3000` will bind port `3000` on the container to `3002` on your machine. So it doesn't matter if `3000` is taken.

**Comment by OhaDerErste** [comment]
oops. Thanks!

**Comment by OhaDerErste** [comment]
Everything seems to work now. I really like the UI, thanks

**Comment by Xyphyn** [comment]
You might get an error that crashes the server. I think i fixed this in v0.9.19
